aria/cli/commands/services.py

```diff
@@ -192,17 +192,17 @@ def outputs(service_name, model_storage, logger):
     """
     logger.info('Showing outputs for service {0}...'.format(service_name))
     service = model_storage.service.get_by_name(service_name)
-    #TODO fix this section..
-    outputs_def = service.outputs
-    response = model_storage.service.outputs.get(service_name)
-    outputs_ = StringIO()
-    for output_name, output in response.outputs.iteritems():
-        outputs_.write(' - "{0}":{1}'.format(output_name, os.linesep))
-        description = outputs_def[output_name].get('description', '')
-        outputs_.write('     Description: {0}{1}'.format(description,
-                                                         os.linesep))
-        outputs_.write('     Value: {0}{1}'.format(output, os.linesep))
-    logger.info(outputs_.getvalue())
+
+    if service.outputs:
+        outputs_ = StringIO()
+        for output_name, output in service.outputs.iteritems():
+            outputs_.write(' - "{0}":{1}'.format(output_name, os.linesep))
+            outputs_.write('     Description: {0}{1}'.format(output.description,
```

Comment:
You can put `os.linesep` in the same line as `Description`, it is short enough, and consistent with the other lines.